org.jacoco.report/src/org/jacoco/report/xml/XMLFormatter.java

```diff
@@ -32,7 +32,7 @@
  */
 public class XMLFormatter {
 
-	private static final String PUBID = "-//JACOCO//DTD Report 1.0//EN";
+	private static final String PUBID = "-//JACOCO//DTD Report 1.1//EN";
```

Comment:
FYI @marchof I triggered works on changing SonarQube integration from reading of binary to XML - https://jira.sonarsource.com/browse/MMF-1362 And for the time being seems that no changes will be required on JaCoCo side. Was thinking that maybe with increasing adoption of XML report, we'll need to introduce something like checksum of source file so that consumer can check it, however as of now I'm not aware of problems with mismatches in case of imports from other coverage tools.

Reply:
Agreed. Can we handle this in a separate PR once someone comes up with a real use case?

Reply:
This was not request of change / addition - just wanted to share thought with you. Let's handle this later when / if we'll see emerging need for real 😉